README.md

```diff
@@ -51,9 +51,39 @@ If you are using a default parser, version in your application and the JSON file
 }
 ```
 
+or
```

Comment:
I do not like this. Please write one example with all possible attributes and in the description, explain which of them are optional.